rollup/bundleHelpers.js

```diff
@@ -102,6 +102,15 @@ export function makeBaseBundleConfig(options) {
     external: builtinModules,
   };
 
+  const workerBundleConfig = {
+    output: {
+      format: 'esm',
```

Comment:
won't esm mess up the CJS bundle?

Reply:
This config is only to build the worker and we only bundle the worker script as ESM because CJS doesn't appear to be supported when injecting the worker code via a data url. Injecting the worker code like this is only supported in Node 16+